tests/test_faker.py

```diff
@@ -123,6 +123,65 @@ def test_months_past(self, write_row_mock):
         assert (date.today() - the_date).days > 80
         assert (date.today() - the_date).days < 130
 
+    def test_date_times(self, generated_rows):
+        with open("tests/test_datetimes.yml") as yaml:
+            generate(yaml)
+
+        for dt in generated_rows.table_values("Contact", field="EmailBouncedDate"):
+            assert "+0" in dt or dt.endswith("Z"), dt
+            assert dateparser.parse(dt).tzinfo
```

Comment:
What about using `dateparser.isoparse()` instead? `parse` is pretty liberal about what it accepts.

Reply:
09f197b